src/redux/categories/categories.js

```diff
@@ -0,0 +1,21 @@
+export const CHECK_STATUS = 'CHECK_STATUS';
+
+export const initialState = {
+  status: [],
+};
```

Comment:
[REQUIRED] Kindly make this an empty array, not an object

Reply:
Hi @fobadara, can you give a little explanation of what you mean by this should be an array, not an object.